http/server.ts

```diff
@@ -122,6 +122,26 @@ export class ServerRequest {
   conn: Conn;
   r: BufReader;
   w: BufWriter;
+  private _cookie: object;
+
+  get cookie(): object {
+    if (this._cookie) {
+      return this._cookie;
+    }
+    if (this.headers.has("Cookie")) {
+      let out: object = {};
```

Comment:
```suggestion
      const out: object = {};
```

Reply:
seem typescript do not recommend using `object` as a type.

try `{[key:string]: string}`?

Reply:
Yes this seems more TypeScript oriented. Nice hint.